app/models/validations/shared_validations.rb

```diff
@@ -54,14 +54,15 @@ def validate_numeric_step(record)
       field = question.check_answer_label || question.id
       incorrect_accuracy = (value.to_d * 100) % (question.step * 100) != 0
 
-      if question.step < 1 && incorrect_accuracy
-        record.errors.add question.id.to_sym, I18n.t("validations.shared.numeric.nearest_hundredth", field:)
-      elsif incorrect_accuracy || value.to_d != value.to_i    # if the user enters a value in exponent notation (eg '4e1') the to_i method does not convert this to the correct value
```

Comment:
N.B. I don't this this `value.to_d != value.to_i` check is actually doing anything useful - anything entered as e.g. "4e1" is being caught by the numeric format validation, and that's a more relevant error message than these.